tool/FileFolderExampleAddIn/FileFolderExampleAddInAddIn.qml

```diff
@@ -0,0 +1,316 @@
+        }
+
+        // makePath
+        // Ref: https://developers.arcgis.com/qt/extras/api-reference/qml-esri-arcgisextras-filefolder.html#makePath-method
+        Row{
+            Layout.fillWidth: true
+
+            Text{
+                id:makePathText
+                text: "makePath:"
+                anchors.left: parent.left
+                font {
+                    family: addIn.font.family
+                    pointSize: 20
+                }
+            }
+
+            TextField  {
+                id: makePathTextField
+                anchors.right: makePathButton.left
+                anchors.left: makePathText.right
+                wrapMode: Text.WrapAnywhere
+                text:legacyStorageFolderPath
+                Layout.fillWidth: true
+
+                font {
+                    family: addIn.font.family
+                    pointSize: 20
+                }
+
+            }
+
+            Button {
+                id: makePathButton
+                text: "Create!"
+                anchors.right: makePathStatusIndicator.left
+                Layout.preferredWidth: parent.width * 0.1
+                onClicked: {
+                    if(makePathFileFolder.makeFolder(makePathTextField.text)){
+                        makePathStatusIndicator.color = "green"
+                    }else{
+                        makePathStatusIndicator.color = "red"
+                    }
+                }
+            }
+
+            Rectangle {
+                id: makePathStatusIndicator
+                anchors.right: parent.right
+                radius: 100
+                color: "white"
+                width: 45
+                height: 45
+            }
+
+        }
+
+        Item {
+            Layout.fillHeight: true
+        }
+
+        Item{
+            Layout.fillWidth: true
+            Layout.minimumHeight: 5
+            Rectangle{
+                anchors.fill: parent; color: "#1167b1"
+            }
+        }
+
+        // File info
+        // ref: https://developers.arcgis.com/qt/extras/api-reference/qml-esri-arcgisextras-fileinfo.html
+
+        Row{
+            Layout.fillWidth: true
+
+            Text{
+                id:fileInfoText
+                text: "makePath:"
+                anchors.left: parent.left
+                font {
+                    family: addIn.font.family
+                    pointSize: 20
+                }
+            }
+
+            TextField  {
+                id: fileInfoTextField
+                anchors.right: parent.right
+                anchors.left: fileInfoText.right
+                wrapMode: Text.WrapAnywhere
+                text:legacyStorageFolderPath
+                Layout.fillWidth: true
+
+                font {
+                    family: addIn.font.family
+                    pointSize: 20
+                }
+
+                onTextChanged: {
+                    fileInfoFileFolder.path = text
+
+                    if(fileInfoFileFolder.exists){
```

Comment:
make sure your code is formatted throughout. In Creator, go to Tools > QML/JS > Reformat File to apply standard formatting